api/test/controllers/users.js

```diff
@@ -0,0 +1,89 @@
+import { expect, request } from 'chai';
+
+import api from '../../src/server';
+import User from '../../src/models/user';
+import { loadFixture, getMockClient, getMockFeed } from '../../src/utils/test';
+import { withLogin } from '../utils';
+
+describe('User controller', () => {
+  let authUser = {
+    email: 'test_user@email.com',
+    password: 'testuser',
+  };
+
+  describe('remove user', () => {
+    let user;
+
+    beforeEach(async () => {
+      await loadFixture('users');
+      user = await User.findOne({email: 'test_user@email.com'});
+      expect(user).to.not.be.null;
+    })
+
+    afterEach(async () => {
+      await User.remove().exec();
+    });
+
+    describe('valid request', () => {
+      let user;
+      let response;
+
+      beforeEach(async () => {
+        user = await User.findOne({email: 'test_user@email.com'});
```

Comment:
Why are we fetching user again here? User is set in `beforeEach` on line 19, but here we fetch it again.

Reply:
Yep - a leftover from earlier test structure.